packages/react-native-audio-api/common/cpp/audioapi/jsi/AudioArrayBuffer.cpp

```diff
@@ -0,0 +1,16 @@
+#include <audioapi/jsi/AudioArrayBuffer.h>
+
+namespace audioapi {
+
+AudioArrayBuffer::AudioArrayBuffer(float *data, size_t size)
+    : data_(data), size_(size) {}
```

Comment:
Just FYI, C++ constructors with empty method body, only with argument list, are usually implemented in header file. Similarly, we could inline the implementation of `size()` and `data()` methods by moving them to the header file as well. However, in such case all methods of `AudioArrayBuffer` would be inline so there would be no key function (pure non-line function) and there could be some problems with RTTI across shared library boundary, i.e. if you create `AudioArrayBuffer` in `libfoo.so`, then `dynamic_cast` would not work correctly in `libbar.so`, it would just always return `null`. So perhaps it's better to have `size()` and `data()` in .cpp file.